lib/resty/kafka/ringbuffer.lua

```diff
@@ -46,6 +64,17 @@ function _M.add(self, topic, key, message)
 end
 
 
+function _M.release_buffer_wait(self, num)
```

Comment:
num is useless? it could only always be 1.

Reply:
> num is useless? it could only always be 1.

ok